markov.py

```diff
@@ -0,0 +1,94 @@
+from collections import Counter
+from bs4 import BeautifulSoup
+import requests
+import string
+from numpy.random import choice
+
+"""
+Importing honor code
+"""
+html = BeautifulSoup(requests.get('http://www.olin.edu/academic-life/student-affairs-resources/student-life/honor-code/').text, 'lxml')
+startpoint = html.get_text().find('The Olin Honor Code Values')
+stoppoint = html.get_text().find('Quick Links')
+
+honor = html.get_text()[startpoint:stoppoint]
+listhonor = str(honor).split()
+
+#print(listhonor)
+
+"""
+Importing felonies
+"""
+html = BeautifulSoup(requests.get('https://en.wikipedia.org/wiki/Felony').text, 'lxml')
+startpoint = html.get_text().find('Broadly, felonies')
+stoppoint = html.get_text().find('are the least serious')
+
+felonies = html.get_text()[startpoint:stoppoint]
+listfelonies = str(felonies).split()
+
+#print(felonies.replace('[',' ').replace(']', ' '))
+
+"""
+Import Olin Promotional Text
+"""
+
+html = BeautifulSoup(requests.get('http://www.olin.edu').text, 'lxml')
+startpoint = html.get_text().find('At Olin')
+stoppoint = html.get_text().find('institutions.')
+
+promotion = html.get_text()[startpoint:stoppoint]
+listpromotion = str(promotion).split()
+
+#print(listpromotion)
+
+"""
+Combing felonies and honor into listhonor
+"""
+
+megalist = []
+megalist.extend(listhonor)
+megalist.extend(listfelonies)
+megalist.extend(listpromotion)
+#print(megalist)
+
+#Finding word frequency
+counts = Counter(megalist)
+#print(counts)
+
+"""Make dictionary
+"""
+def markov(megalist):
+
+
+    i = 0;
+    markovdict = {}
+    for i in range(len(megalist)-1):
+        if megalist[i] not in markovdict:
+            markovdict[megalist[i]] = []
+        markovdict[megalist[i]].append(megalist[i+1])
```

Comment:
This is pretty complex operation going on here and as a reader, it is hard to understand what's going on. In-line comment describing what is happening would be good.